### ISSUE TITLE
Metrics and cross validation

### ISSUE THREAD

**Issue by julianikulski** [julianikulski/data-science-resources#7]
**Resource**: https://scikit-learn.org/stable/modules/cross_validation.html#cross-validation
**Description**: 3.1. Cross-validation: evaluating estimator performance

**Resource**: https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.cross_val_score.html
**Description**:  sklearn.model_selection.cross_val_score

**Resource**: http://scikit.ml/
**Description**:  Multi-Label Classification in Python

**Resource**: https://stackoverflow.com/questions/61640817/f1-score-for-multiclass-labeling-cross-validation
**Description**:  F1 score for multiclass labeling cross validation

**Resource**: https://stats.stackexchange.com/questions/225949/do-we-need-a-test-set-when-using-k-fold-cross-validation
**Description**: Do we need a test set when using k-fold cross-validation?
**Details**: This discussion explains that there should in almost all cases be a test set separated from the train-validation set that is never changed and never runs through the training or validation phase of the model.
